.github/workflows/repo-checks-tests.yml

```diff
@@ -74,6 +74,7 @@ jobs:
       REDIS_PORT: 6379
       REDIS_PASSWORD: redis
       REDIS_STANDALONE_MODE: "true"
+      WORKERS_PORT: 3002
```

Comment:
Do we need this as we have a default value for repo checks?

Reply:
changed to 3020 default port